internal/config/config.go

```diff
@@ -55,6 +56,7 @@ type Config struct {
 	BundlePath            string                  `config:"bundle_path"`
 	PackagePolicyId       string                  `config:"package_policy_id"`
 	PackagePolicyRevision int                     `config:"revision"`
+	Index                 string                  `config:"index"`
```

Comment:
Our integration has the [correct ECS `data_stream` fields](https://www.elastic.co/guide/en/ecs/current/ecs-data_stream.html) [defined](https://github.com/elastic/integrations/blob/317769902bd73c80c9de621f364d2de6547c2322/packages/cloud_security_posture/data_stream/findings/fields/base-fields.yml), and we could use those to build the index, like so `{data_stream.type}-{data_stream.dataset}-{data_stream.namespace}`.
However, the `namespace` field is empty for current release of the integration. The Policy/Config also comes with the `index` field, which contains a ready-to-use index name in the correct format. 

@kfirpeled Do you know where does `index` field come from and is it reliable enough for us to use?

Reply:
How I understand is that a integration has the permissions to to publish to a specific index. See the policy template once you install it. So essentially the integration already knows where you can publish to. The index template is decided by the integration

Reply:
Yeah, that I know :) I wanted to know if there is anything preventing us from relying on the `index` field.

Reply:
I'm not familiar with `index` in the agent config. I'm only familiar with agent's policy.

Here's a snippet:

```yaml
inputs:
  - id: logfile-system-1c2284c7-c5ae-481d-a7bc-d9e3ce61545a
    name: system-3
    revision: 1
    type: logfile
    use_output: default
    meta:
      package:
        name: system
        version: 1.58.1
    data_stream:
      namespace: default
    package_policy_id: 1c2284c7-c5ae-481d-a7bc-d9e3ce61545a
    streams:
      - id: logfile-system.auth-1c2284c7-c5ae-481d-a7bc-d9e3ce61545a
        data_stream:
          dataset: system.auth
          type: logs
```

So I can only guess from here how it is being done on the beats side